lib/ansible/module_utils/eos.py

```diff
@@ -419,16 +425,17 @@ def load_config(self, config, commit=False, replace=False):
 
         return result
 
-is_json = lambda x: str(x).endswith('| json')
-is_text = lambda x: not str(x).endswith('| json')
 
-supports_sessions = lambda x: get_connection(module).supports_sessions
+def is_json(cmd):
+    return str(cmd).endswith('| json')
```

Comment:
`cmd `  is the command that needs to run on device eg. `show version | json`. 
It is given as input in task.
If the input command is bad it is not expected to fail as this point hence changed the line to.
```to_native(cmd, errors='surrogate_then_replace').endswith('| json')```